frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -42,4 +44,36 @@ describe('Localization', () => {
     cy.visit('/dashboards?pseudolocalization=true&lng=en');
     cy.byTestID('utilization-card-item-text').isPseudoLocalized();
   });
+
+  it('pseudolocalizes monitoring pages', () => {
+    const monitoringPages = ['alerts', 'silences', 'alertrules'];
+    monitoringPages.forEach((page) => {
+      cy.visit(`/monitoring/${page}?pseudolocalization=true&lng=en`);
+      if (page === 'alerts' || page === 'alertrules') {
+        listPage.rows.shouldBeLoaded();
+        cy.get(ListPageSelector.tableColumnHeaders).isPseudoLocalized();
+        cy.get(DetailsPageSelector.horizontalNavTabs).isPseudoLocalized();
+      }
+      // check applied filters are i18ned then clear all filters
+      cy.get('.pf-c-chip-group__main').then((groups) => {
+        if (groups.length >= 1) {
+          cy.get('[class*="pf-c-chip-group"]').isPseudoLocalized();
```

Comment:
Hi, after digging into this code (and others we have), this approach isn't correct.  The `cy.get` returns the text from all of the chips (in this example) and does one call to `.isPseudoLocalized()` on the entire combined string! 

I am going to work on a PR today which fixes this in our existing i18n tests

I believe you want to do this:
```
cy.get('[class*="pf-c-chip-group"]').each(($el) => {
   cy.wrap($el).isPseudoLocalized();
});
```